docs/learn/learn-xcvm.md

```diff
@@ -39,12 +118,77 @@ enum Instruction {
 }
 ```
 
-A `MultiAsset` is a general identifier for an asset. It may represent both fungible and non-fungible
-assets, and in the case of a fungible asset, it represents some defined amount of the asset.
+- A `MultiAsset` is a general identifier for an asset. It may represent both fungible and
+  non-fungible assets, and in the case of a fungible asset, it represents some defined amount of the
+  asset.
+
+- A `MultiLocation` is a relative identifier, meaning that it can only be used to define the the
+  relative path between two locations, and cannot generally be used to refer to a location
+  universally.
+
+`TransferAsset` is one of the many instructions that can be contained within an XCM. For more
+information, please read [XCM Instructions in the wiki](./learn-xcm-instructions.md).
+
+## Locations in XCM
```

Comment:
It might be prudent to mention the Universal Location in its own subsection here.

Reply:
@gavofyork Added - I'm not entirely sure it's accurate, so let me know if anything needs to be changed! If it's all well, I'll add a graphic to demonstrate a scenario.

As a sidenote, this concept will really come in handy when explaining how a bridge between global networks could actually look like 👍